contents/python/introduction-to-python.md

```diff
@@ -12,6 +12,15 @@ For absolute beginners, check out [this video](https://www.youtube.com/watch?v=N
 
 Both newbies and experienced programmers can also benefit from [The Python Tutorial](https://docs.python.org/3/tutorial/index.html), which aims to introduce readers to Python's unique features and style.
 
+## Virtual Environment
+
+When starting out Python projects, you will be recommended to install dependencies using [virtual environment](https://docs.python.org/3/tutorial/venv.html) to remove the possibility of dependencies conflict. This is especially a good practice when managing dependencies from different projects which relies on different python versions and packages.
```

Comment:
What exactly is meant by `starting out python projects`? Is it when you are creating a new project from scratch or when your are setting up a pre-existing project on your device. This part confused me a bit so maybe it can be rephrased

Reply:
It can be for both. But it is good to mention both cases specifically. Thanks for pointing out!